.vscode/settings.json

```diff
@@ -1,9 +1,9 @@
 {
   "css.lint.validProperties": ["composes"],
   "editor.codeActionsOnSave": {
-    "source.organizeImports": false,
-    "source.fixAll.eslint": true,
-    "source.fixAll.shellcheck": true
+    "source.organizeImports": "false",
```

Comment:
I'm still seeing changes to this file…there shouldn't be any changes to it, as it's unrelated to the rest of the PR